.github/workflows/wheels.yml

```diff
@@ -157,3 +157,56 @@ jobs:
           # https://anaconda.org/multibuild-wheels-staging/numpy
           # The tokens were originally generated at anaconda.org
           upload_wheels
+  build_sdist:
+    name: Build sdist
+    needs: get_commit_message
+    if: >-
+      contains(needs.get_commit_message.outputs.message, '[wheel build]') ||
+      github.event_name == 'schedule' ||
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && ( ! endsWith(github.ref, 'dev0')))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout numpy
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          # versioneer.py requires the latest tag to be reachable. Here we
+          # fetch the complete history to get access to the tags.
+          # A shallow clone can work when the following issue is resolved:
+          # https://github.com/actions/checkout/issues/338
+          fetch-depth: 0
+      # Used to push the built wheels
+      - uses: actions/setup-python@v2
+        with:
+          # Build sdist on lowest supported Python
+          python-version: '3.8'
+      - name: Build sdist
+        run: |
+          pipx run build --version
+          pipx run build --sdist
+      - name: Test the sdist
+        run: |
+          python -m pip install dist/*.gz
+          python -c "import numpy; pandas.show_versions();"
```

Comment:
This may be a silly question, but why are we using `pandas` to test the source distribution of NumPy built here?

Reply:
My bad. I was porting the sdist build scripts from pandas over here to the numpy repo. Sorry for the silly mistake and thanks for the catch.